NewsHSE/Modules/News/NewsViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewController.swift
+//  NewsHSE
+//
+//  Created by Darya Zhitova on 07.11.2023.
+//
+
+import UIKit
+
+protocol NewsDisplayLogic: AnyObject {
+    func displayNews(_ viewModel: NewsDataFlow.Presentation.ViewModel)
+}
+
+final class NewsViewController: UIViewController, NewsDisplayLogic {
+    
+    lazy var contentView: NewsView = {
```

Comment:
Как подпишешь NewsView под протокол, не забудь здесь поменять.
```suggestion
    lazy var contentView: DisplaysNewsView = {
```
Да, хоть вьюха и контроллер могут быть жестко связаны , все равно стоит закрыть протоколом, чтобы лишний раз обеспечить сокрытие.

Reply:
хорошо) подскажи пожалуйста, мне получается нужно все покрывать протоколами?

Reply:
Даа. Обмазываемся протоколами по полной. Модельки не стоит. Хотя у нас в проекте были попытки.))

Reply:
модели я не хотела протоколить, ахахахахахах

хотя на работе так делала